src/app/pages/account/update/index.js

```diff
@@ -0,0 +1,25 @@
+/* ============
+ * Account Index Page
+ * ============
+ *
+ * Page where the user can view the account information
+ */
+import accountService from '../../../services/account'
+
+
+export default {
+ data() {
+    return {
+      user: this.$store.state.account
```

Comment:
This isn't gonna work. Because you are binding the user using `v-model`, you will get the error that you are not supposed to update the `user` outside of the store.
See the [docs](https://vuex.vuejs.org/en/forms.html#twoway-computed-property) for more information